### PR TITLE
[KOGITO-4664] Add PMMLs with scorecard and mining model to examples

### PR DESCRIPTION
**Pull request by jiripetrlik**

### CODE REVIEW

**Review by gitgabrio — COMMENTED** [review]
@jiripetrlik Thanks for that PR. I'm wondering if it would be better to also add "wrong data" tests: wrong data may be different on a by-model base, so covering them could give a better monitor on what is going on. WDYT ?

File: dmn-pmml-quarkus-example/src/test/java/org/kie/pmml/kogito/quarkus/example/NativeDecisionTreeIT.java [hunk 1]
```diff
@@ -18,7 +18,7 @@
 import io.quarkus.test.junit.NativeImageTest;
 
 @NativeImageTest
-public class NativeSampleMineIT extends SampleMineTest {
+public class NativeDecisionTreeIT extends DecisionTreeTest {
```
Comment (danielezonca) [inline dmn-pmml-quarkus-example/src/test/java/org/kie/pmml/kogito/quarkus/example/NativeDecisionTreeIT.java:21]: Native test must follow `Native*IT` naming convention

**Comment by jiripetrlik** [comment]
@gitgabrio Yes, you are right. I've added tests for wrong data to all models in Spring boot using previously developed tools.

**Review by gitgabrio — APPROVED** [review]
Thanks @jiripetrlik LGTM

**Review by danielezonca — APPROVED** [review]

**Pull request merged**